src/morphocut/image.py

```diff
@@ -311,6 +311,10 @@ def after_stream(self):
         print("Absolute: ", min(self.min), max(self.max))
         print("Average: ", mean_min, mean_max)
 
+        ImageDraw.text(image, '{}mm'.format(self.scale_size), fill=None, font=None, anchor=None, spacing=4, align='left', direction=None,
```

Comment:
Please create a new Node `ImageCaption`:

```python
@...
class ImageCaption(Node):
  ...
  def transform(self, image):
    ...
    return image_with_caption
```

Also, you don't have to supply all parameters if you don't change the default value.